torch/_meta_registrations.py

```diff
@@ -113,7 +113,9 @@ def checkUplo(uplo: str):
     ), f"Expected UPLO argument to be 'L' or 'U', but got {uplo}"
 
 
-@torch.library.impl(meta_lib, "linalg_eigh")
+# Keeping this meta impl around, but we don't want to register it directly to the meta key
+# because `aten::linalg_eigh` is composite.
+# `_linalg_eigh` is implemented internally as a structured kernel, so we have meta support.
```

Comment:
we might need a python meta function anyways, for the purposes of symbolic shape propagation.